.circleci/config.yml

```diff
@@ -346,10 +375,16 @@ workflows:
       #    PYTEST_ARGS: --numprocesses=1
       # SPHINCS exhausts memory on CircleCI servers
       # for these configurations.
-      - arm_emulated:
+      # don't run ARM64 emulated any more:
+      #- arm_emulated:
+      #    <<: *require_buildcheck
+      #    name: arm64
+      #    ARCH: arm64
+      #    CMAKE_ARGS: -DOQS_ENABLE_SIG_SPHINCS=OFF -DOQS_DIST_BUILD=ON
+      - arm_machine:
           <<: *require_buildcheck
           name: arm64
-          ARCH: arm64
+          PYTEST_ARGS: --ignore=tests/test_portability.py --numprocesses=auto --maxprocesses=10
```

Comment:
We should run test_portability. We don't have any code dispatching on ARM yet, but we will soon.

Reply:
Done with update